src/ttt_core/console.clj

```diff
@@ -59,50 +65,73 @@
 (defn draw-main [game message]
   (do
     (clear-terminal)
-    (draw-header)
+    (draw-header (:language game))
     (draw-player-info game)
     (draw-board (:board game))
     (println message)
     (draw-footer)))
 
-(defn build-choose-game-mode-string 
-  ([] (build-choose-game-mode-string ""))
-  ([message]
-    (str message "Choose a game mode:\n1. Human-vs-Human\n2. Human-vs-Computer")))
-
-(defn request-game-mode
-  ([game] (request-game-mode game ""))
-  ([game message]
-    (let [modes {:1 :human-vs-human :2 :human-vs-computer}]
+(defn build-selection-string 
+  ([language msg-keyword] (build-selection-string language msg-keyword ""))
+  ([language msg-keyword prepend-msg]
+    (str prepend-msg
+      (if (= "" prepend-msg) "" " ")
```

Comment:
It might be slightly more readable if you didn't have this conditional here and just ran your final message through a function that strips leading whitespace.

Reply:
Addressed.